Utilities/bin_patch.cpp

```diff
@@ -427,6 +430,11 @@ bool patch_engine::add_patch_data(YAML::Node node, patch_info& info, u32 modifie
 
 	switch (p_data.type)
 	{
+	case patch_type::utf8:
+	{
+		p_data.value.long_value = p_data.original_value.size();
```

Comment:
I think this line can be removed, as long_value is not used anywhere for utf8

Reply:
hmm, right, it's unused